### PR TITLE
docs: add Image Style Props examples

### PR DESCRIPTION
**Pull request by ronnyiscoding**
Issue #1579 Add Image Style Props examples.

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -341,9 +341,6 @@
       "toastandroid": {
         "title": "ToastAndroid"
       },
-      "toolbarandroid": {
```
Comment (Simek) [inline website/i18n/en.json]: Am I assuming correctly that this change was added to the commit by mistake?

Comment (ronnyiscoding) [reply website/i18n/en.json]: Yes, I will remove it

Comment (Simek) [reply website/i18n/en.json]: So it seems that someone forget to add a change generated after building app to their PR which landed on `master`. Regardless that this change should not be a part of this PR in my opinion.

Comment (ronnyiscoding) [reply website/i18n/en.json]: PR update https://github.com/facebook/react-native-website/pull/1602/commits/8d87b86c9943022a26b5e9bba6bead7c173f8c6b changes restored

**Comment by rachelnabors** [comment]
@ronnyiscoding My last concern: have you updated the API?

**Pull request merged**